README.md

```diff
@@ -8,6 +8,26 @@ Download the repository to your computer:
 git clone https://github.com/cta-observatory/2022_01_lstchain_school.git
 ```
 
+## Install
+
+- You will need to install [miniconda](https://docs.conda.io/en/latest/miniconda.html) (recommended) or [anaconda](https://www.anaconda.com/distribution/#download-section) first. 
+
+
+### In your computer
+
+```
+conda env create -f environment.yml
+conda activate lst-school
+```
+
+### In the IT cluster
+
+```
+source /fefs/aswg/software/conda/etc/profile.d/conda.sh
+conda env create -f environment.yml
```

Comment:
Should we really let everyone create their own personal school env in the cluster?

Than we should tell them to at least put the conda env in workspace, not their `$HOME` via following the instructions here:

https://www.lst1.iac.es/wiki/index.php/Working_at_the_LST-IT_center

Reply:
no, I think you are right.
I included this just for whoever wanted to create their own personal school env in the cluster, but if people see it here, then many are going to install it there, so I just removed it